src/dashboard/DashboardView.react.js

```diff
@@ -101,10 +102,8 @@ export default class DashboardView extends React.Component {
         link: '/push/new'
       });
     }
-    // The push UI requires immediate and scheduled push (and some ruby endpoints that we will have to remove)
-    /*
 
-    if (features.push && features.push.storedPushData) {
+    if (features.push && parseServerVersion >= '2.2.8') {
```

Comment:
We are trying to avoid inspecting the server version directly, and instead adding info to the `serverInfo` endpoint in Parse Server to let the dashboard know that new features are available. Could you do that instead?

This particular check would also start to fail once we get to `2.10.0` or etc.